compose-dev.yaml

```diff
@@ -0,0 +1,12 @@
+services:
```

Comment:
You should remove this file from git (such as deleting it, lmk if that breaks anything)